CIP-0164/README.md

```diff
@@ -1937,7 +1937,46 @@ lower right)</em>
 
 </div>
 
-<a name="resource-requirements"></a>**Resource requirements**
+##### Behavior under varying load
+
+The simulation results also provide evidence that Leios operates as a well-behaved protocol, conforming to three specific desirable properties.
+
+1. *Throughput is proportional to load until it plateaus when capacity is reached:* Figure 14a illustrates the near equality of throughput to load up until the approximately 300 TxkB/s capacity (determined by the protocol parameters) is reached, beyond which throughput stays constant. (Note that the randomness of sortition and the finite duration of the simulation results in some jitter and uncertainty in the data points even though the trend is clear.) It is particularly important that the throughput does not degrade at very high demand: instead Leios provides backpressure on clients, via the memory pool, so that the protocol operates at capacity even though demand is higher.
+2. *Transaction delivery time and its variance remains constant up until the protocol's capacity is reached:* The left side of Figure 14b demonstrates that the observed range of transaction delivery time, which is defined in these plots as the time from being submitted to the time of reaching the ledger, stays essentially constant nearly until capacity is reached. Because the Leios simulator has an unbounded (unlimited) memory pool, one sees that times increase beyond that due to the memory pool growing ever larger. Near capacity, fluctuations from sortitition result in uncertainty. The right side of Figure 14b shows the additional time imposed by Leios relative to Praos: at very low demand, Leios does not often forge EBs and the delay is small, but at moderate and high loads a nearly constant delay is associated with most transactions appearing in EBs. This property is important in that Leios performance does not degrate as demand approaches or exceeds capacity.
+3. *The processing cost per transaction decreases or levels off as load increases:* Figure 14c evidences that the CPU and network used per transaction generally drops as load (i.e., demand) increases. (Once again there are uncertainties in this plot due to the randomness or sortition and design choices in the Leios simulator.) This property is important in that the per-transaction consumption of resources should not lower efficiency, which would lead to disproportional stress on resources as load increases.
+
+<div align="center">
+<a name="figure-14a" id="figure-14a"></a>
```

Comment:
You already asked about how to re-generate the figure index on slack. @will-break-it is there a script for that? I just ran `ouroboros-leios/scripts/format-cip.sh` on this file and it would only do markdown formatting, but no index of figures and tables.

Do we need those indices?